_posts/2016-09-29-Rust-1.12.md

```diff
@@ -0,0 +1,305 @@
+---
+layout: post
+title: "Announcing Rust 1.12.0"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.12.0. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+As always, you can [install Rust 1.12.0][install] from the appropriate page on our
+website, and check out the [detailed release notes for 1.12.0][notes] on GitHub.
+1361 patches were landed in this release.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/stable/RELEASES.md#version-1120-2016-09-29
+
+### What's in 1.12.0 stable
+
+Oh my god! So much stuff!
```

Comment:
Ha!  I think brson put this in as a joke.  I don't mind something fun here since there is a lot of stuff in this release, but maybe "Oh my god!" is a step too far.